examples/gno.land/p/demo/blog/blog.gno

```diff
@@ -287,15 +354,9 @@ type Comment struct {
 }
 
 func (c Comment) RenderListItem() string {
-	output := ""
-	output += ufmt.Sprintf("#### %s\n", formatAuthorAndDate(c.Author, c.CreatedAt))
+	output := "" // todo see where it is used
+	// output += ufmt.Sprintf("#### %s\n", formatAuthorAndDate(c.Author, c.CreatedAt))
```

Comment:
This should now be fixed on master

Reply:
I brushed up the comment section a bit: [6c32b09](https://github.com/gnolang/gno/pull/1541/commits/6c32b094273e4941fe26ba6369995a7d333bbce9)